docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md

```diff
@@ -0,0 +1,90 @@
+# Uploading data and model checkpoints to s3
+There are two ways to upload them: 
+1. using python script
```

Comment:
by using python script you use the command line interface. I believe you wanted to write Application Programming Interface(API)